components/Bounty/SuccessScreen.tsx

```diff
@@ -0,0 +1,123 @@
+import { css, StyleSheet } from "aphrodite";
+import Image from "next/image";
+import { ReactElement, useState } from "react";
+import icons from "~/config/themes/icons";
+
+type Props = {
+  bountyAmount: number;
+  bountyText: string;
+  postId: number;
+  postSlug: string;
+};
+
+function SuccessScreen({
+  bountyAmount,
+  bountyText,
+  postId,
+  postSlug,
+}: Props): ReactElement {
+  const twitterPreText = `Help me with my bounty on ResearchHub worth ${bountyAmount} RSC!`;
+  const [copySuccess, setCopySuccess] = useState(false);
+
+  const link = process.browser
+    ? window.location.protocol +
+      "//" +
+      window.location.hostname +
+      `/post/${postId}/${postSlug}`
+    : "";
+
+  const twitterBountyPreview = `\n\n"${bountyText.slice(
+    0,
+    249 - twitterPreText.length - link.length
+  )}"\n\n${link}`;
+
+  function copyToClipboard() {
```

Comment:
Can we create a util for this? maybe a file in `config/util/copyToClipboard.ts`?
I see this function being used frequently and think it would be nice to reference it once rather than copy/paste.